project/Dependencies.scala

```diff
@@ -5,93 +5,109 @@ object Dependencies {
   val sprayV = "1.3.3"
   val slickV = "3.1.1"
 
+  val googleV = "1.22.0"
+  val olderGoogleV = "1.20.0"   // TODO why do we have two google versions?
```

Comment:
Not specifically.  Looks like the newer ones were just added more recently.  We could add a tech debt story to update/align these.